applications/common/backend/src/main/resources/libs/kyanite/common/components/table/tablecell/tablecell.html

```diff
@@ -17,7 +17,7 @@
 */-->
 
 <sly data-sly-use.model="pl.ds.kyanite.common.components.models.table.TableCellComponent">
-  <td rowspan="${model.rows}" colspan="${model.columns}">
+  <td rowspan="${model.rowspan}" colspan="${model.colspan}">
```

Comment:
Actually, we call computation of colspan here (getColspan() calls computeColspan())